scripts/evidence/test_release.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import print_function
+
+import subprocess
+import os
+import json
+import time
+
+from sdk.actions import (
+    ListValidators,
+    ByzantineFault_Requests,
+    NodeID,
+    GetBlockHeight,
+    GetFrozenMap,
+)
+
+from sdk.rpc_call import (
+    node_0,
+    node_1,
+    node_2,
+    node_3,
+)
+
+from sdk.cmd_call import (
+    GetNodeCreds,
+    Account_Add,
+    ByzantineFault_Allegation,
+    Send,
+    GetNodeKey,
+    ByzantineFault_Vote,
+    KillNode,
+)
+
+from base import valDict, malicious
+
+
+def test_release():
+    print("Starting release test")
+    # checking if validators is not frozen
+    fMap = GetFrozenMap()
+    assert len(fMap) == 0, 'Frozen validator found'
+
+    is_killed = KillNode(node_1)
+    assert is_killed is True, 'Failed to kill node %s' % node_1
+
+    # freezeing height and waiting 4 blocks
+    height = GetBlockHeight()
+    check_height = height + 4
+    print("Waiting height %d to proceed (current: %s)" % (
+        check_height, height,
+    ))
+    while check_height >= height:
+        height = GetBlockHeight()
+        time.sleep(1)
+
+    print("Height %s ready" % check_height)
+
+    # checking if validator is frozen
+    fMap = GetFrozenMap()
+    assert valDict['1']['address'] in fMap, 'Validator %s not frozen' % malicious
+
+    print("Test for release successfully passed!")
+
+
+if __name__ == "__main__":
+    test_release()
```

Comment:
Can we add functionality for testing release ? It just tests freezing here

Reply:
If it freezing than something with the node. Could you give a more details?

Reply:
I mean after it has been frozen , can we also send a release transaction ,in the test .
 - Create a secure wallet account using validator pub/priv key 
 - Send a release request 
 - wait to check if Frozen becomes False

Reply:
it is covered by tests already, see the code

Reply:
What I mean is , follow the following steps 
make reset 
python scripts/evidence/test_release.py
olclient validatorset 

- the last command should show it as Frozen = False , 
For that to happen you need to send a release transaction from the test  , olclient byzantine_fault release

Reply:
Can you clarify, not clear

Reply:
you need to add some code , to send a release transaction ,from the frozen validator ,at the end

Reply:
this test related to killing a node and check if it frosted as it not match a vote criteria at N of blocks. For real releasing it better be implemented as a separate test.